doc/releases/changelog-0.29.0.md

```diff
@@ -501,7 +503,7 @@
 
 <h4>Operations and batching</h4>
 
-* A new function called `qml.dot` has been added to compute the dot product between a vector and a list of operators.
+* The function `qml.dot` has been updated to compute the dot product between a vector and a list of operators.
```

Comment:
I am not sure we had it before?

Reply:
Yes it was there but for collections: https://docs.pennylane.ai/en/stable/code/api/pennylane.dot.html